nullaway/src/main/java/com/uber/nullaway/fixserialization/Serializer.java

```diff
@@ -195,4 +203,116 @@ public static Path pathToSourceFileFromURI(@Nullable URI uri) {
       return path;
     }
   }
+
+  /**
+   * Serializes the given {@link Symbol} to a string.
+   *
+   * @param symbol The symbol to serialize.
+   * @return The serialized symbol.
+   */
+  public static String serializeSymbol(@Nullable Symbol symbol) {
+    if (symbol == null) {
+      return "null";
+    }
+    switch (symbol.getKind()) {
+      case FIELD:
+      case PARAMETER:
+        return symbol.name.toString();
+      case METHOD:
+      case CONSTRUCTOR:
+        return serializeMethodSignature((Symbol.MethodSymbol) symbol);
+      default:
+        return symbol.flatName().toString();
+    }
+  }
+
+  /**
+   * Serializes the signature of the given {@link Symbol.MethodSymbol} to a string.
+   *
+   * @param methodSymbol The method symbol to serialize.
+   * @return The serialized method symbol.
+   */
+  private static String serializeMethodSignature(@Nullable Symbol.MethodSymbol methodSymbol) {
```

Comment:
I don't think `methodSymbol` needs to be `@Nullable`

Reply:
Sure, left over from previous implementation. Removed it e6ad24d94ff4fce7e874411696064b8070f2edcc.